cves/kernel/CVE-2018-13100.yml

```diff
@@ -147,10 +143,10 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: A developer reported a bug on 2018-06-22, alongside sharing a crafted F2FS image which triggered the crash. They were not a Google employee.
```

Comment:
Was there anything written about what they were doing at the time of the discovery?  If the issue is exploited through improper image files, was the F2FS image used at discovery corrupted?